portal-api/src/v1/gef/controllers/facilities.controller.js

```diff
@@ -42,7 +42,7 @@ exports.create = async (req, res) => {
       res.status(201).json(response);
     }
   } else {
-    res.status(422).send([{ errCode: 'MANDATORY_FIELD', errMsg: 'No Application ID and/or facility type sent with request' }]);
+    res.status(422).send([{ status: 'MANDATORY_FIELD', errMsg: 'No Application ID and/or facility type sent with request' }]);
```

Comment:
@abhi-markan are we sure we want to make this change? `MANDATORY_FIELD`, `company-profile-not-found`, `ENUM_ERROR`, etc. do seem like error codes rather than statuses to me

Reply:
Good spot, can we do following please.

```javascript
{ status: 422, errCode: 'MANDATORY_FIELD'....}
```